### ISSUE TITLE
[Feat. request] Headset-only 'seated play' support

### ISSUE THREAD

**Issue by dashwr** [Sorenon/MCXR#53]
**TL;DR - Headset-only as in no motion controllers, locking the crosshair to the middle of the screen or something similar.**

Some people can't afford proper VR and thus do hacky stuff with Google Cardboard headsets. It would be pretty awesome if this mod was compatible with a seated headset + keyboard and mouse setup, so that playing on Google Cardboard was made viable.

I'm nearly sure Vivecraft supports that, but it's kinda odd and literally unplayable on an iGPU (Intel's OpenGL drivers have shoddy OpenGL VR support, afaik) so I'm hoping MCXR gets that feature since it seems really promising + fabric performance mods FTW.

You can achieve an unplayable (as in unable to interact with anything) but still cool seated setup in MCXR by combining it with a controller via LambdaControls/MidnightControls, and that's pretty much the only reason I'm requesting proper support.

**Comment by wellsilver** [comment]
Yeah that's not happening unless you do it yourself

**Comment by Flamgop** [comment]
> Yeah that's not happening unless you do it yourself, I'm sure it's simple enough though just pojav launcher and looking

Sir, this is MCXR not questcraft

**Comment by dashwr** [comment]
> Yeah that's not happening unless you do it yourself, I'm sure it's simple enough though just pojav launcher and looking

What? Even if i used Pojav, that feature still isn't in MCXR. And even if it was, making use of it on a phone would be nearly impossible.
Besides, i play Minecraft on my computer, not my phone.

**Comment by Sorenon** [comment]
When the new input system is done this should be easily doable

**Comment by NathanVH8** [comment]
> When the new input system is done this should be easily doable

Any idea of roughly when that might be?
